README.md

```diff
@@ -1,2 +1,3 @@
 # sturdy-umbrella
+
```

Comment:
Check out your commits: in two commits you have wrong credentials and also one of them have incorrect commit message style (starts with lower case)